experimenter/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -140,6 +144,27 @@ export const DirectoryColumnEndDate: ColumnComponent = ({
   </td>
 );
 
+export const DirectoryColumnUnpublishedUpdates: ColumnComponent = ({
+  publishStatus: p,
+}) => (
+  <td data-testid="directory-table-cell">
+    {p ? (
+      <>
+        <Badge
+          className={
+            "ml-2 border rounded-pill px-2 bg-white font-weight-normal border-danger text-danger"
+          }
+          data-testid="directory-unpublished-updates"
+        >
+          {p === NimbusExperimentPublishStatusEnum.DIRTY ? "YES" : ""}
```

Comment:
Actually what we should do is add one of those aggregate states to that status object that computes the aggregate set of states that have 'unpublished changes' and then we can use that on the summary page pill and here.  But we can do that as a followup.